code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -166,53 +163,69 @@ You should now be able to use your Explode verb to interface with the nuclear fi
 	return
 
 
-/datum/game_mode/malfunction/proc/takeover()
-	set category = "Malfunction"
-	set name = "System Override"
-	set desc = "Start the victory timer"
-
+/spell/aoe_turf/takeover
+	name = "System Override"
+	panel = MALFUNCTION
+	desc = "Start the victory timer"
+	charge_type = Sp_CHARGES
+	charge_max = 1
+	
+/spell/aoe_turf/takeover/before_target(mob/user)
 	if (!istype(ticker.mode,/datum/game_mode/malfunction))
 		to_chat(usr, "<span class='warning'>You cannot begin a takeover in this round type!</span>")
-		return
+		return 1
 	if (ticker.mode:malf_mode_declared)
 		to_chat(usr, "<span class='warning'>You've already begun your takeover.</span>")
-		return
+		return 1
 	if (ticker.mode:apcs < 3)
 		to_chat(usr, "<span class='notice'>You don't have enough hacked APCs to take over the station yet. You need to hack at least 3, however hacking more will make the takeover faster. You have hacked [ticker.mode:apcs] APCs so far.</span>")
-		return
+		return 1
 
 	if (alert(usr, "Are you sure you wish to initiate the takeover? The station hostile runtime detection software is bound to alert everyone. You have hacked [ticker.mode:apcs] APCs.", "Takeover:", "Yes", "No") != "Yes")
-		return
-
+		return 1
+		
+/spell/aoe_turf/takeover/cast(var/list/targets, mob/user)
 	command_alert(/datum/command_alert/malf_announce)
 	set_security_level("delta")
 
 	ticker.mode:malf_mode_declared = 1
 	for(var/datum/mind/AI_mind in ticker.mode:malf_ai)
-		AI_mind.current.verbs -= /datum/game_mode/malfunction/proc/takeover
-
-
-/datum/game_mode/malfunction/proc/ai_win()
-	set category = "Malfunction"
-	set name = "Explode"
-	set desc = "Station goes boom"
-
+		for(var/spell/S in AI_mind.current.spell_list)
+			if(istype(S,src))
```

Comment:
It really wouldn't surprise me if it does work because BYOND does lots of silly undocumented things, but yeah it should be `type` even if `src` does work just for the sake of doing things how they *should* be done